docs/pages/getting-started/developers.md

```diff
@@ -124,7 +124,7 @@ You can also email us directly at uswebdesignstandards@gsa.gov.
 * Uses a **[modified BEM](https://pages.18f.gov/frontend/css-coding-styleguide/naming/)** approach created by 18F for naming CSS selectors. Objects in CSS are separated by single dashes. Multi-word objects are separated by an underscore (For example: `.usa-button-cool_feature-active`).
 * Uses **modular CSS** for scalable, modular, and flexible code.
 * Uses **nesting** when appropriate. Nest minimally with up to two levels of nesting.
-* **Global variables** are defined in the `_defaults.scss` file. Custom theming can be done by copying `_variables.scss`, changing variable values and importing it before uswds in Sass.
+* **Global variables** are defined in the `_defaults.scss` file. Custom theming can be done by copying `_variables.scss` into your own project's Sass folder, changing variable values, and importing it before uswds in Sass.
```

Comment:
@kategarklavs perhaps! what we're trying to say is they can change whichever variables apply to them for their project's needs. Does "relevant" infer that?

Also what do you think about modifying "copying `_variables.scss`" slightly to: 
>copying the `_variables.scss` file...